### PR TITLE
fix(interactive): Introduce `Degree Fusion` RBO Rules

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
fuse the following situations to expandDegree + sum:
1. expand+getV+count
2. expand+count

## Related issue number

Fixes

### CODE REVIEW

File: interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/planner/rules/DegreeFusionRule.java [hunk 1]
```diff
@@ -0,0 +1,244 @@
+package com.alibaba.graphscope.common.ir.planner.rules;
```
Comment (longbinlai) [inline interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/planner/rules/DegreeFusionRule.java:1]: Copyright?

**Review by longbinlai — COMMENTED** [review]
Can we manage a doc page for all these optimization rules?

File: interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/planner/rules/DegreeFusionRule.java (hunk 1 above)
Comment (shirly121) [reply interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/planner/rules/DegreeFusionRule.java:17]: done

**Comment by shirly121** [comment]
> Can we manage a doc page for all these optimization rules?

done

**Pull request merged**